bundles/model/org.eclipse.smarthome.model.sitemap/src/org/eclipse/smarthome/model/Sitemap.xtext

```diff
@@ -36,7 +36,8 @@ Text:
 		('visibility=[' (Visibility+=VisibilityRule (',' Visibility+=VisibilityRule)* ']'))?);
 
 Group:
-	'Group' (('item=' item=GroupItemRef) & ('label=' label=(ID | STRING))? & ('icon=' icon=Icon)? &
+	'Group' (('item=' item=GroupItemRef) & ('label=' label=(ID | STRING))? & ('icon=' icon=Icon)? & 
+		('expended=' expended=Expended)? &
```

Comment:
```(expanded?='expanded')?``` would be nicer, than you do not have to provide true or false at all (and thus you can remove the Expended (+Boolean) rule.